cmd/ddev/cmd/composer-create-project.go

```diff
@@ -248,7 +248,43 @@ ddev composer create-project --prefer-dist --no-interaction --no-dev psr/log .
 			})
 
 			if err != nil {
-				util.Failed("Failed to install project: %v\nstderr=%v", err, stderr)
+				util.Warning("Failed to install project: %v\nstderr=%v", err, stderr)
+				// Try to run composer update instead.
+				output.UserOut.Println("Trying 'composer update' instead of 'composer install'...")
```

Comment:
I was worried that this could be overreach, but since it's only on `composer create-project` it's probably OK.